### PR TITLE
Register reviewer/participant: website => api => db

### PR DESCRIPTION
**Pull request by JustD4nTe**
- Create endpoints for reviewer and participant to login and register
- Write participant avatar on server
- After successful register move to main page

### CODE REVIEW

File: web-client/src/components/SignUpForm.jsx [hunk 1]
```diff
@@ -141,17 +146,23 @@ export default function SignUpForm(props) {
   const handleChangeSelect = (event) => {
     setSpecialization(event.target.value);
   };
+
   const style = styles();
 
+  // store reference/data of form
+  const formRef = React.useRef(null);
```
Comment (Kwolik) [inline web-client/src/components/SignUpForm.jsx:153]: We don't do that anymore? 
We just define (React.useRef) at the beginning

**Review by Kwolik — CHANGES_REQUESTED** [review]

**Review by Kwolik — APPROVED** [review]

**Review by KamilDonda — APPROVED** [review]

**Review by upsu90 — CHANGES_REQUESTED** [review]
I found a bug: after adding a photo and pressing "Sign up" nothing happens (even after deleting the photo) :c

**Comment by JustD4nTe** [comment]
> I found a bug: after adding a photo and pressing "Sign up" nothing happens (even after deleting the photo) :c

I forgot to include a special folder for files like user's photo :C Last commit should fix this bug

**Review by upsu90 — APPROVED** [review]
everything works!

**Pull request merged**